packages/vscode-extension/package.json

```diff
@@ -1614,6 +1614,7 @@
     "test-watch": "tsc -watch -p ./",
     "pretest": "npm run lint && npm run check-format && npm run test-compile",
     "test:unit": "rimraf out && rimraf coverage && npm run compile && nyc mocha --config .mocharc.json \"test/**/*.test.ts\"",
+    "test:lifecycle": "rimraf out && rimraf coverage && npm run compile && nyc mocha --config .mocharc.json \"test/handlers/lifecycleHandlers.test.ts\"",
```

Comment:
Is this line needed or just for testing when preparing PR?

Reply:
Just for test